src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs

```diff
@@ -85,6 +85,26 @@ public StringValues(string?[]? values)
             return value.GetArrayValue();
         }
 
+#if NET9_0_OR_GREATER
```

Comment:
@dotnet/area-extensions-primitives please advice if you want to expose this to ns2.0 too, it can't be allocation-free there.